src/multiselectMixin.js

```diff
@@ -591,7 +605,7 @@ export default {
       }
 
       const index = typeof option === 'object'
```

Comment:
I think we could get rid of that check since `getOptionKey` now handles a case where it just needs to return the plain value.